packages/debridge/src/debridge.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { JsonRpcProvider, Wallet } from 'ethers';
+import { Debridge } from './debridge';
+import { describe, it, expect } from 'vitest';
+import { DeBridgeErrorCodes } from './debridge.types';
+import { tokens } from './tokens';
+import { Url } from '@gardenfi/utils';
+import { createWalletClient, http } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { mainnet } from 'viem/chains';
+
+describe('DeBridge', () => {
+  const debridge = new Debridge(
+    new Url('/v1.0', 'https://api.dln.trade'),
```

Comment:
take strings instead of Url object.
Url object should be created inside the constructed.